docs/libcurl/opts/CURLOPT_HTTP_VERSION.md

```diff
@@ -88,6 +88,14 @@ given in the URL, with fallback to earlier HTTP versions if needed.
 server given in the URL and does not downgrade to earlier HTTP version if the
 server does not support HTTP/3.
 
+## CURL_HTTP_VERSION_3_V2
```

Comment:
I don't think we should overload the HTTP version option with QUIC details. If we really need to specifically ask for QUIC versions (which I doubt) then we should probably add a new option for that.